presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -75,6 +75,9 @@
     @Option(name = "--user", title = "user", description = "Username")
     public String user = System.getProperty("user.name");
 
+    @Option(name = "--password", title = "password", description = "Password")
```

Comment:
sure. Do you want me to add a new CLI option called --enable-ldap and prompt for password only when it is set, since we want passwords to be optional?